be/src/exec/tablet_sink.cpp

```diff
@@ -689,8 +689,8 @@ Status OlapTableSink::send_chunk(RuntimeState* state, vectorized::Chunk* chunk)
         if (num_rows_after_validate - _validate_select_idx.size() > 0) {
             size_t filtered_size = num_rows_after_validate - _validate_select_idx.size();
             std::string debug_row = chunk->debug_row(invalid_row_index);
-            state->append_error_msg_to_file(
-                    debug_row, strings::Substitute("there are $0 rows couldn't find a partition", filtered_size));
+            state->append_error_msg_to_file(debug_row,
+                                            "The row is out of partition ranges. Please add a new partition.");
```

Comment:
If the change of the code is not complicated, it is best to show which partitionkey is missing.

Reply:
Under this framework, printing the specified column will be difficult. Instead, I print the row directly.